### PR TITLE
Log autoencoder results to wandb.

### PR DESCRIPTION
**Pull request by vladfi1**

### CODE REVIEW

File: gym_sts/autoencoder/train.py [hunk 1]
```diff
@@ -234,15 +262,27 @@ def train():
                 results=results, prefix=f"Batch: {batch_num+1}/{total_batches}"
             )
 
+            to_log = dict(
+                loss=results["loss"].double(),
+                top1=results["top1"].double(),
+                epoch=epoch + batch_num / total_batches,
```
Comment (kronion) [inline gym_sts/autoencoder/train.py:268]: Is this meant to be a float value? halfway through epoch 1, the value should be 1.5?

File: gym_sts/autoencoder/train.py [hunk 2]
```diff
@@ -234,15 +262,27 @@ def train():
...
 
-    for epoch in range(10):
-        print(f"Epoch {epoch}")
-        train()
+        to_log = dict(
+            loss=val_results["loss"].double(),
+            top1=val_results["top1"].double(),
+            epoch=epoch + 1,
+        )
+        wandb.log(dict(valid=to_log), step=train_step)
```
Comment (kronion) [inline gym_sts/autoencoder/train.py]: Could we be more specific, `validation=`?

**Review by kronion — APPROVED** [review]
Accepted with comments

File: gym_sts/autoencoder/train.py (hunk 1 above)
Comment (vladfi1) [reply gym_sts/autoencoder/train.py:268]: Yes that's right.

File: gym_sts/autoencoder/train.py (hunk 2 above)
Comment (vladfi1) [reply gym_sts/autoencoder/train.py]: Done

**Pull request merged**